<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optiuniInsert</class>
 <widget class="QDialog" name="optiuniInsert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>403</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>403</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Insert Optiuni</string>
  </property>
  <widget class="QComboBox" name="nrMatricolBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NR. MATRICOL:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="codCaminBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>COD CAMIN:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nrCameraBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NR. CAMERA:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="coleg1Box">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>COLEG 1:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="coleg3Box">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>COLEG 3:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="coleg2Box">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>COLEG 2:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okInsert">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelInsert">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
